### PR TITLE
[DNM] Fish Farming, KobeChem, Hunger + Sound Combination PR

### PR DESCRIPTION
**Pull request by coldud13**
## About The Pull Request
Just adding both to one PR to fix conflictssssss

## Changelog
N/A

### CODE REVIEW

File: modular_lumos/code/game/machinery/fish_tanks.dm [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+		new /obj/structure/fish_tank(get_turf(src))
+		qdel(src)
+		return
+	else if(istype(I, /obj/item/screwdriver))
+		I.play_tool_sound(src, 50)
+		if(!do_after(user, 2 SECONDS, FALSE, src))
+			return
+		I.play_tool_sound(src, 50)
+		var/obj/item/stack/sheet/metal/metal = new /obj/item/stack/sheet/metal(get_turf(src))
+		metal.amount = 20
```
Comment (EvenInDeathIStillServe) [inline modular_lumos/code/game/machinery/fish_tanks.dm:26]:
```suggestion
		metal.amount = 10
```
Construction needs 10 metal. You currently end up with 10 duplicated metal on disassembly.

File: modular_lumos/code/game/objects/items/fish_fish.dm [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+/obj/item/fishy/attackby(obj/item/I, mob/living/user, params)
+	if(I.get_sharpness())
+		to_chat(user, "<span class='notice'>You begin to butcher [I]...</span>")
+		playsound(I.loc, 'modular_skyrat/sound/effects/butcher.ogg', 50, TRUE, -1)
+		if(!do_after(user, 4 SECONDS, FALSE, src))
+			return
+		for(var/spawned_meat in meat)
+			for(var/i in 1 to rand(1,3))
+				new spawned_meat(get_turf(src))
+		new /obj/effect/gibspawner/generic(get_turf(src))
```
Comment (EvenInDeathIStillServe) [inline modular_lumos/code/game/objects/items/fish_fish.dm:141]:
```suggestion

```
I've played around with it, and when you start chopping up more than one fish, on a table, nonetheless, the whole kitchen ends up looking like a scene out of Cannibal Holocaust.

File: modular_lumos/code/game/machinery/fish_tanks.dm [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+		new /obj/structure/fish_tank_base(get_turf(src))
+		var/obj/item/stack/sheet/glass/glass = new /obj/item/stack/sheet/glass(get_turf(src))
+		glass.amount = 5
+		qdel(src)
+		return
+	else if(istype(I, /obj/item/fish_tool/fish_food))
+		var/obj/item/fish_tool/fish_food/food = I
+		if(food.food_left >= 5)
+			food.food_left -= 5
+			current_food += 5
```
Comment (EvenInDeathIStillServe) [inline modular_lumos/code/game/machinery/fish_tanks.dm:110]:
```suggestion
			current_food += 10
```
I feel that fish food is consumed too fast. I think doubling the quantity inserted would be a good idea.

File: modular_lumos/code/game/machinery/fish_tanks.dm [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+			food.desc = "[initial(food.desc)] Food Remaining: [food.food_left]"
+			playsound(src.loc, 'sound/machines/chime.ogg', 50, TRUE, -1)
```
Comment (EvenInDeathIStillServe) [inline modular_lumos/code/game/machinery/fish_tanks.dm:112]: I don't think this sound is indicative of pouring fish food into a tank.

**Review by EvenInDeathIStillServe — CHANGES_REQUESTED** [review]
Hi, first Github review ever so please bear with me. I played around with fish culture a little and these are my thoughts about improvements. I can see that this is just a compatibility PR and might not be interested in making these changes, I understand. Nonetheless here is what I think, I might make an issue or even make changes myself later on.
One thing that irked me is that the fish are all normal-sized items, and they can't be stored on trays. My suggestion would be to make them small-sized and make them able to be picked up by trays and, for lack of a better container, plant bags.
Lobsters seem to be neglected in the cooking department. Their only edible forms are raw, and they are not included in any recipes. Even deepfrying does not change the fact that they're raw.
The fish themselves get processed way too fast. Even if you only have one or two tanks, you'll need to constantly babysit them lest the fish die or get too old to reproduce.

File: modular_lumos/code/game/machinery/fish_tanks.dm (hunk 1 above)
Comment (jjpark-kb) [reply modular_lumos/code/game/machinery/fish_tanks.dm]: fixed

File: modular_lumos/code/game/objects/items/fish_fish.dm (hunk 2 above)
Comment (jjpark-kb) [reply modular_lumos/code/game/objects/items/fish_fish.dm]: fixed/removed

File: modular_lumos/code/game/machinery/fish_tanks.dm (hunk 4 above)
Comment (jjpark-kb) [reply modular_lumos/code/game/machinery/fish_tanks.dm:112]: the tank chimes duh! "I have no good sound for it"

File: modular_lumos/code/game/machinery/fish_tanks.dm (hunk 3 above)
Comment (jjpark-kb) [reply modular_lumos/code/game/machinery/fish_tanks.dm:110]: fixed/done

**Comment by coldud13** [comment]
This PR has been inactive for long enough to be automatically marked as stale. This means it is at risk of being auto closed in ~ 14 days, please address any outstanding review items and ensure your PR is finished, if these are all true and you are auto-staled anyway, you need to actively ask maintainers if your PR will be merged. Once you have done any of the previous actions then you should request a maintainer remove the stale label on your PR, to reset the stale timer. If you feel no maintainer will respond in that time, you may wish to close this PR youself, while you seek maintainer comment, as you will then be able to reopen the PR yourself

**Pull request closed without merging**